packages/spacebars-compiler/templatetag.js

```diff
@@ -210,6 +218,9 @@ TemplateTag.parse = function (scannerOrString) {
       return ['STRING', result.value];
     } else if (/^[\.\[]/.test(scanner.peek())) {
       return ['PATH', scanPath()];
+    } else if (/^\(/.test(scanner.peek())) {
```

Comment:
I feel like this should be `scanner.run(/\(/)` instead of a peek and then pos += 1